hazelcast/src/main/java/com/hazelcast/internal/bootstrap/ClientMain.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.bootstrap;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.map.IMap;
+
+import static com.hazelcast.internal.bootstrap.TpcServerBootstrap.ALTO_ENABLED;
+import static com.hazelcast.internal.bootstrap.TpcServerBootstrap.ALTO_EVENTLOOP_COUNT;
+
+/**
+ * Demo application for TPC. Will be removed in in the final release.
```

Comment:
todo: remove and create a code sample instead

Reply:
Hmm, yes but this is very useful for dev process. We can discuss what to do with this but we can remove it from core as well.